src/plugins/links/public/editor/open_editor_flyout.tsx

```diff
@@ -148,5 +148,8 @@ export async function openEditorFlyout({
     if (overlayTracker) {
       overlayTracker.openOverlay(editorFlyout);
     }
+  }).catch(() => {
```

Comment:
Instead of adding catch, just change `reject` in `onClose` to `resolve(undefined)`.

Change 
```
const onCancel = () => {
      reject();
      closeEditorFlyout(editorFlyout);
    };
```

To 
```
const onCancel = () => {
      resolve(undefined);
      closeEditorFlyout(editorFlyout);
    };
```

Reply:
I did this to keep things consistent with `src/plugins/links/public/editor/open_link_editor_flyout.tsx` and that is how we are doing it there - but I'm fine with this, too! I can make that change to both files.

Reply:
Done in 18a1b83ccc044b0501995c80bf807e3810deac13 :+1: